### ISSUE TITLE
Nextflow doesn't use Service Account Credentials on the GCP instance 

### ISSUE THREAD

**Issue by snamburi3** [nextflow-io/nextflow#1693]
## Bug report 

The GCP instance started by Cloud Life Sciences / Nextflow uses the Default service account on the instances. Nextflow does not seem to use the service account credentials to download the data. Could you tell us what credentials are used by Nextflow and the mechanism?

### Expected behavior and actual behavior

We have a service account (default in this case) approved to access data stored in a cross-project bucket. The expected behaviour with Cloud Life Sciences is that the service account credentials are used to access the data (needs roles/iam.serviceAccountUser role). Nextflow doesn't seem to use service account credentials. 
If it does, could you please let us know how to use it with gsutil?

### Program output 

```
gsutil -m -u PROJECT_ID" cp ${bam_path} .
```
`ServiceException: 401 Requester pays bucket access requires authentication.`

### Environment 

* Nextflow version: 20.07.1

### Additional context

Use cases:
1. Cross Project bucket access with service accounts
2. Long running jobs in production need to use service account credentials

**Comment by pditommaso** [comment]
Not sure to understand from where that message comes from. Can you include the steps to replicate the problem?

**Comment by snamburi3** [comment]
the message is from the `gsutil` tool. We managed to use gsutil in nextflow pipeline. we had to use container `google/cloud-sdk:alpine` in the process. Custom install of gcloud/gsutil didnt work - maybe because of our configuration. Here is the code for others who need to use gsutil in their pipeline for various reasons.

```
process download {
container google/cloud-sdk:alpine
input FILE_NAME
output *.vcf into process
command:
gsutil -m -u PROJECTNAME cp FILE_NAME .
}
```

this issue can be closed.
